### PR TITLE
Fix/Refactor Clone Channel action

### PR DESCRIPTION
**Pull request by tlfn**
**Please describe the changes this PR makes and why it should be merged:**
Discord.js `GuildChannel#clone` doesn't accept position.
Delete some duplicates

**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [ ] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: actions/clone_channel_MOD.js [hunk 1]
```diff
@@ -157,40 +157,40 @@ module.exports = {
...
+      .then(async (newChannel) => {
+        if (data.position === 1) {
+          await newChannel.setPosition(channel.position);
+        }
+        const storage2 = parseInt(data.storage2, 10);
+        const varName2 = this.evalMessage(data.varName2, cache);
+        this.storeValue(newChannel, storage2, varName2, cache);
+        this.callNextAction(cache);
+      })
+      .catch(this.displayError.bind(this, data, cache));
```
Comment (almeidx) [inline actions/clone_channel_MOD.js:193]: This doesn't call the next action

**Review by almeidx — CHANGES_REQUESTED** [review]

File: actions/clone_channel_MOD.js (hunk 1 above)
Comment (tlfn) [reply actions/clone_channel_MOD.js:193]: It didn't, should it?

Comment (almeidx) [reply actions/clone_channel_MOD.js:193]: Yes

**Comment by tlfn** [comment]
@almeidx Bump

**Review by almeidx — APPROVED** [review]

**Pull request merged**